libs/dh/shared/data-access-mocks/src/lib/wholesale.ts

```diff
@@ -612,21 +610,6 @@ function getGridAreasQuery() {
   });
 }
 
-function getLatestBalanceFixing() {
-  return mockGetLatestBalanceFixingQuery(async () => {
-    await delay(mswConfig.delay);
-    return HttpResponse.json({
-      data: {
-        __typename: 'Query',
-        latestBalanceFixing: {
-          __typename: 'Calculation',
-          period: { start: periodStart, end: periodEnd },
-        },
-      },
-    });
-  });
-}
```

Comment:
suggestion: one out. Remember to add one as well :) Add mock for GetLatestCalculation.